h/templates/deform/form.jinja2

```diff
@@ -5,39 +5,50 @@
 {% set form_buttons_class = 'form-actions-buttons' %}
 {% set form_message_class = 'form-actions-message' %}
 {% endif %}
-
+{% if field.use_inline_editing %}
+{% set form_controller_class = 'js-form' %}
+{% endif %}
 <form id="{{ field.formid }}"
       action="{{ field.action }}"
       method="{{ field.method }}"
       enctype="multipart/form-data"
       accept-charset="utf-8"
-      class="form {% if field.css_class %}{{ field.css_class }} {% endif %}">
+      class="form {{ field.css_class or '' }} {{ form_controller_class }}">
```

Comment:
Personally I think this is kind of icky. It's a bit nicer to keep the conditionals where they're used and use Jinja's whitespace control to keep the output reasonable:

```jinja2
<form ...
     class="form
            {%- if field.css_class %} {{ field.css_class}}{% endif -%}
            {%- if field.use_inline_editing %} js-form{% endif -%}
     ">
```